src/interfaces/IERC7540.sol

```diff
@@ -181,6 +181,18 @@ interface IERC7540CancelRedeem {
         returns (uint256 shares);
 }
 
+interface IAuthorizeOperator {
```

Comment:
```suggestion
interface IAuthorizedOperator {
```

Reply:
The method is `authorizeOperator` so ithink the current name makes sense. But this will be replaced by an ERC number in the future anyways